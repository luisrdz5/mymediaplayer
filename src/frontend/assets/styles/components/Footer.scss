.blue {
    background-color: #5630fb;
}

.footer{
  //  background-color: #21C08B;
    display:flex;
    align-items: center;
    height: 100px;
    width: 100%;
}
.footer a{
    color: white;
    cursor: pointer;
    font-size: 14px;
    padding-left: 30px;
    text-decoration: none;
}
.footer a:hover{
    text-decoration: underline;
}