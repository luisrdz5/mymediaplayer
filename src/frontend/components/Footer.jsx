import React from 'react';
import '../assets/styles/components/Footer.scss';
import classNames from 'classnames';

const Footer = (props) => {
  console.log(props);
  const { isLogin } = props;
  let headerClass;
  if (isLogin) {
    headerClass = classNames('footer blue');
  } else {
    headerClass = classNames('footer');
  }
  console.log(headerClass);
  return (
    <footer className={headerClass}>
      <a href='./'>Terminos de uso</a>
      <a href='./'>Declaraci√≥n de Privacidad</a>
      <a href='./'>Centro de Ayuda</a>
    </footer>
  );

};
export default Footer;
