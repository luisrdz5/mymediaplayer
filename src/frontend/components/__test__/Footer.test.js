import React from 'react';
import { create } from 'react-test-renderer';
import Adapter from 'enzyme-adapter-react-16';
import { render, shallow, configure } from 'enzyme';
import Footer from '../Footer';

configure({ adapter: new Adapter() });

describe('Footer testing', () => {
    test('Match Snapshot', () => {
        const footer = create(<Footer />);
        expect(footer.toJSON()).toMatchSnapshot();
    });
    test('footer has class .footer', () => {
        const footer = shallow(<Footer />);
        const footerElem = footer.find('footer');
        expect(footerElem.hasClass('footer')).toBe(true);
    });
    test('footer has 3 anchor tags', () => {
        const footer = render(<Footer />);
        expect(footer.find('a')).toHaveLength(3);
    });
});