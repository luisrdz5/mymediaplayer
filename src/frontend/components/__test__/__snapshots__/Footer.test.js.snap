// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer testing Match Snapshot 1`] = `
<footer
  className="footer"
>
  <a
    href="./"
  >
    Terminos de uso
  </a>
  <a
    href="./"
  >
    Declaraci√≥n de Privacidad
  </a>
  <a
    href="./"
  >
    Centro de Ayuda
  </a>
</footer>
`;
